pages/AWorkedExample.html

```diff
@@ -34,7 +34,7 @@ <h4>First Card</h4>
     Click on the Content box under "First Card" that you saw when the Exploration
     Editor first opened, and type in the following:
   </p>
-  <img src="static/images/tutorial/card1InitialContent.png" class="tutorial-image">
+  <img src="static/images/tutorial/card1InitialContent.png" class="tutorial-image" alt="Content for Card 1">
```

Comment:
Here and elsewhere, I believe alt text is supposed to describe the image for screenreader users. If you write "Content for Card 1" or "View of the Card 1 after number input" then the user won't know how to visualize that.

Consider perhaps more visual descriptions of the images, such as: "A textarea showing the content for card 1, with 'Save Content' and 'Cancel' buttons below it", or "A blue box showing the interaction, followed by an 'Oppia Responses' section with a single 'All answers' field."\

See perhaps https://blog.hubspot.com/marketing/image-alt-text and other resources.

Reply:
The alt-text is indeed supposed to describe the image for users who are unable to view it. I did not emphasize on the user-perspective before, but after the reference provided by you I have done the needful. I apologise for not having taken this into consideration previously. I have accounted for the requested changes, do let me know if there are anymore discrepancies, I'd be more than happy to oblige!